packages/bigquery/test/test.js

```diff
@@ -6,10 +6,11 @@ let results;
 
 test('query runs', async () => {
     try{
-        results = await runQuery("select 100 as num")
-        assert.instance(results, Array)
-        assert.type(results[0], "object")
-        assert.equal(results[0].num, 100)
+        results = await runQuery("select 100 as num");
```

Comment:
Would it make sense to expand our tests to cover all of the native types? E.g. 

```suggestion
        results = await runQuery("select 100 as number, current_date() as date, current_timestamp() as timestamp, 'Evidence' as string");
```